src/swap/components/SwapButton.tsx

```diff
@@ -5,12 +5,15 @@ import type { SwapButtonReact } from '../types';
 
 export function SwapButton({
   className,
+  onError,
+  onSuccess,
   disabled = false,
```

Comment:
Always keep alphabetical order with props.

Reply:
I thought default values should be last? at least in some languages if there is a default value the style guide indicates they have to be last in the ordered parameters, but that may be python

Reply:
fixed in new diff